packages/ngtools/webpack/src/resource_loader.ts

```diff
@@ -200,6 +222,14 @@ export class WebpackResourceLoader {
 
           parent.warnings.push(...childCompilation.warnings);
           parent.errors.push(...childCompilation.errors);
+          for (const { info, name, source } of childCompilation.getAssets()) {
+            if (info.sourceFilename === undefined) {
+              throw new Error(`'${name}' asset info 'sourceFilename' is 'undefined'.`);
+            }
+
+            parent.updateAsset(name, source, info);
```

Comment:
I don't think this update call is needed.  The assets are emitted with essentially the same loop right before this callback is executed.  If there is any overlap with the cached assets, there should be an error before this is updated.
https://github.com/webpack/webpack/blob/master/lib/Compiler.js#L526-L528